dashboard/client/src/components/ActorTable.tsx

```diff
@@ -267,6 +270,19 @@ const ActorTable = ({
             <TextField {...params} label="IP" />
           )}
         />
+        <Autocomplete
+          data-testid={"nodeIdFilter"}
```

Comment:
nit: for strings, lets's avoid unnecessary brackets.
```suggestion
          data-testid="nodeIdFilter"
```